<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ActionTemplateEditorDialog</class>
 <widget class="QDialog" name="ActionTemplateEditorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>487</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ChangeMe!</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabInfo">
      <attribute name="title">
       <string>Описание</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <property name="margin">
        <number>4</number>
       </property>
       <property name="spacing">
        <number>4</number>
       </property>
       <item row="3" column="1" colspan="2">
        <widget class="QLabel" name="lblActionType">
         <property name="text">
          <string>&amp;Тип действия</string>
         </property>
         <property name="buddy">
          <cstring>cmbActionType</cstring>
         </property>
        </widget>
       </item>
       <item row="5" column="4" colspan="2">
        <widget class="QComboBox" name="cmbSex">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <item>
          <property name="text">
           <string/>
          </property>
         </item>
         <item>
          <property name="text">
           <string>М</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Ж</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="11" column="0" colspan="5">
        <widget class="QRadioButton" name="rbVisibleToSpeciality">
         <property name="text">
          <string>Специальность</string>
         </property>
        </widget>
       </item>
       <item row="15" column="0" colspan="2">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>51</width>
           <height>331</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="7" column="1">
        <widget class="QLabel" name="lblName_2">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>&amp;Диагнозы</string>
         </property>
         <property name="buddy">
          <cstring>edtMKBList</cstring>
         </property>
        </widget>
       </item>
       <item row="6" column="6">
        <widget class="QLineEdit" name="edtBegAgeCount">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="inputMask">
          <string>000; </string>
         </property>
         <property name="maxLength">
          <number>3</number>
         </property>
        </widget>
       </item>
       <item row="6" column="8">
        <widget class="QComboBox" name="cmbEndAgeUnit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
         </property>
         <item>
          <property name="text">
           <string/>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Д</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Н</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>М</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Г</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="10" column="0" colspan="5">
        <widget class="QRadioButton" name="rbVisibleToOwner">
         <property name="text">
          <string>Врач</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="11" column="5" colspan="5">
        <widget class="CRBComboBox" name="cmbSpeciality">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="6" column="9" colspan="2">
        <widget class="QLineEdit" name="edtEndAgeCount">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="inputMask">
          <string>000; </string>
         </property>
         <property name="maxLength">
          <number>3</number>
         </property>
        </widget>
       </item>
       <item row="6" column="7">
        <widget class="QLabel" name="lblAgeSep">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>по</string>
         </property>
         <property name="buddy">
          <cstring>cmbEndAgeUnit</cstring>
         </property>
        </widget>
       </item>
       <item row="9" column="0" colspan="5">
        <widget class="QRadioButton" name="rbVisibleToOrgStructure">
         <property name="text">
          <string>Подразделение</string>
         </property>
        </widget>
       </item>
       <item row="6" column="4" colspan="2">
        <widget class="QComboBox" name="cmbBegAgeUnit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <item>
          <property name="text">
           <string/>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Д</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Н</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>М</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Г</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="8" column="0" colspan="12">
        <widget class="QLabel" name="lblVisibleTo">
         <property name="text">
          <string>Этот шаблон могут применять:</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="Line" name="line_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="4" colspan="9">
        <widget class="CRBComboBox" name="cmbGroup">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="4" colspan="9">
        <widget class="QLineEdit" name="edtName">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="3" column="4" colspan="9">
        <widget class="CActionTypeComboBox" name="cmbActionType"/>
       </item>
       <item row="5" column="6" colspan="7">
        <spacer>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>231</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="6" column="11" colspan="2">
        <spacer>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>71</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="QLabel" name="lblGroup">
         <property name="text">
          <string>Группа</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="13">
        <widget class="QLabel" name="lblBaseProperties">
         <property name="text">
          <string>Основные свойства шаблона</string>
         </property>
        </widget>
       </item>
       <item row="7" column="4" colspan="9">
        <widget class="QLineEdit" name="edtMKBList">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="2">
        <widget class="QLabel" name="lblAge">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>&amp;Возраст</string>
         </property>
         <property name="buddy">
          <cstring>cmbBegAgeUnit</cstring>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="2">
        <widget class="QLabel" name="lblSex">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>&amp;Пол</string>
         </property>
         <property name="buddy">
          <cstring>cmbSex</cstring>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="13">
        <widget class="QLabel" name="lblCase">
         <property name="text">
          <string>Этот шаблон можно применять в случаях:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="3">
        <widget class="QLabel" name="lblName">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>На&amp;именование</string>
         </property>
         <property name="buddy">
          <cstring>edtName</cstring>
         </property>
        </widget>
       </item>
       <item row="9" column="5" colspan="8">
        <widget class="COrgStructureComboBox" name="cmbOrgStructure"/>
       </item>
       <item row="14" column="0" colspan="13">
        <widget class="QRadioButton" name="rbVisibleToAll">
         <property name="text">
          <string>Все</string>
         </property>
        </widget>
       </item>
       <item row="13" column="0" colspan="13">
        <widget class="QRadioButton" name="rbVisibleToSNILS">
         <property name="text">
          <string>По СНИЛС</string>
         </property>
        </widget>
       </item>
       <item row="11" column="10" colspan="3">
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>232</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="10" column="5" colspan="8">
        <widget class="CPersonComboBoxEx" name="cmbOwner"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabProperties">
      <attribute name="title">
       <string>Свойства</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="margin">
        <number>4</number>
       </property>
       <property name="spacing">
        <number>4</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="lblStatus">
         <property name="text">
          <string>Статус</string>
         </property>
         <property name="buddy">
          <cstring>cmbStatus</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="CActionStatusComboBox" name="cmbStatus">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="lblAmount">
         <property name="text">
          <string>Количество</string>
         </property>
         <property name="buddy">
          <cstring>edtAmount</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QSpinBox" name="edtAmount"/>
       </item>
       <item row="0" column="4">
        <widget class="QLabel" name="lblOffice">
         <property name="text">
          <string>Кабинет</string>
         </property>
         <property name="buddy">
          <cstring>edtOffice</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <widget class="QLineEdit" name="edtOffice">
         <property name="maxLength">
          <number>16</number>
         </property>
        </widget>
       </item>
       <item row="0" column="6">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>107</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lblPerson">
         <property name="text">
          <string>Исполнитель</string>
         </property>
         <property name="buddy">
          <cstring>cmbPerson</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lblStatusNote">
         <property name="text">
          <string>Примечания</string>
         </property>
         <property name="buddy">
          <cstring>edtNote</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="6">
        <widget class="QLineEdit" name="edtNote"/>
       </item>
       <item row="3" column="0" colspan="7">
        <widget class="CActionPropertiesTableView" name="tblProps">
         <property name="enabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="6">
        <widget class="CPersonComboBoxEx" name="cmbPerson">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="4" margin="4"/>
 <customwidgets>
  <customwidget>
   <class>CActionTypeComboBox</class>
   <extends>QComboBox</extends>
   <header>Events/ActionTypeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>CActionPropertiesTableView</class>
   <extends>QTableView</extends>
   <header>Events/ActionPropertiesTable.h</header>
  </customwidget>
  <customwidget>
   <class>CPersonComboBoxEx</class>
   <extends>QComboBox</extends>
   <header>Orgs/PersonComboBoxEx.h</header>
  </customwidget>
  <customwidget>
   <class>CRBComboBox</class>
   <extends>QComboBox</extends>
   <header>library/crbcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>CActionStatusComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">Events/ActionStatus.h</header>
  </customwidget>
  <customwidget>
   <class>COrgStructureComboBox</class>
   <extends>QComboBox</extends>
   <header>Orgs/OrgStructComboBoxes.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>cmbGroup</tabstop>
  <tabstop>edtName</tabstop>
  <tabstop>cmbActionType</tabstop>
  <tabstop>cmbSex</tabstop>
  <tabstop>cmbBegAgeUnit</tabstop>
  <tabstop>edtBegAgeCount</tabstop>
  <tabstop>cmbEndAgeUnit</tabstop>
  <tabstop>edtEndAgeCount</tabstop>
  <tabstop>edtMKBList</tabstop>
  <tabstop>rbVisibleToOrgStructure</tabstop>
  <tabstop>cmbOrgStructure</tabstop>
  <tabstop>rbVisibleToOwner</tabstop>
  <tabstop>cmbOwner</tabstop>
  <tabstop>rbVisibleToSpeciality</tabstop>
  <tabstop>cmbSpeciality</tabstop>
  <tabstop>rbVisibleToAll</tabstop>
  <tabstop>cmbStatus</tabstop>
  <tabstop>edtAmount</tabstop>
  <tabstop>edtOffice</tabstop>
  <tabstop>cmbPerson</tabstop>
  <tabstop>edtNote</tabstop>
  <tabstop>tblProps</tabstop>
  <tabstop>buttonBox</tabstop>
  <tabstop>rbVisibleToSNILS</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>rbVisibleToOwner</sender>
   <signal>toggled(bool)</signal>
   <receiver>cmbOwner</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>61</x>
     <y>288</y>
    </hint>
    <hint type="destinationlabel">
     <x>427</x>
     <y>290</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbVisibleToSpeciality</sender>
   <signal>toggled(bool)</signal>
   <receiver>cmbSpeciality</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>63</x>
     <y>317</y>
    </hint>
    <hint type="destinationlabel">
     <x>322</x>
     <y>335</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ActionTemplateEditorDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>253</x>
     <y>438</y>
    </hint>
    <hint type="destinationlabel">
     <x>243</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ActionTemplateEditorDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>253</x>
     <y>438</y>
    </hint>
    <hint type="destinationlabel">
     <x>243</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbVisibleToOrgStructure</sender>
   <signal>toggled(bool)</signal>
   <receiver>cmbOrgStructure</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>113</x>
     <y>259</y>
    </hint>
    <hint type="destinationlabel">
     <x>287</x>
     <y>261</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
